kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/util/dialog/ErrorDialog.java

```diff
@@ -43,6 +43,7 @@
 public class ErrorDialog extends AlertDialog {
     
     private String debugInfo;
+    private String detailsTitle;
```

Comment:
Good point.  But, even if there would be a default title, one would need to prevent that null is assigned as a new value. So the null check would effectively move into the setter. But indeed , there is no null check for the debugInfo. Nevertheless, I'd consider such things as a separate PR. Thanks for the review!

Reply:
> I'd consider such things as a separate PR

Agreed.

> one would need to prevent that null is assigned as a new value

If someone is passing null via public API, that's their problem 😄